src/Domains/Connectors/ESimGo/Services/ESimGoService.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Kanvas\Connectors\ESimGo\Services;
+
+use Baka\Contracts\AppInterface;
+use Kanvas\Connectors\ESimGo\Client;
+use Kanvas\Exceptions\ValidationException;
+
+class ESimGoService
```

Comment:
@tiberius19 is this the EsimService ?

Reply:
Yes but for Esimgo, you asked to keep them separate, each encapsulated in his own connector